openmicroscopy/openmicroscopy
# Right panel annotation tabs2

This is a follow-up to https://github.com/openmicroscopy/openmicroscopy/pull/4579 with a refactor of tag loading and a lot of other code cleanup / removal.
Also added some web tests for annotate_tags.

To test:
 - Browse all data types in tree
 - Batch annotate Tags, adding & removing Tags
 - NB: This following is all be tested by Robot tests
    - Check that only tags on all selected objects show up in right column of tag dialog
    - Adding tags to right adds to all objects
    - Moving tags to left removes from all objects
 - Descriptions are displayed in tag tooltip and long text wraps:

![screen shot 2016-05-13 at 11 47 33](https://cloud.githubusercontent.com/assets/900055/15250154/4536afe0-191b-11e6-9fe3-cf98ffbbd6b0.png)
